### ISSUE TITLE
Remove python_terminal plugin from the Leo distribution

### ISSUE THREAD

**Issue by gatesphere** [leo-editor/leo-editor#2148]
As the original author of the python_terminal plugin, I'd like to request it be removed from the distro.  It is incredibly buggy, and I do not wish for my bad old code to continue to inflict pain on the Leo community.

I am unable to give it the attention I would need to fix it properly.  It does not accomplish much that can't be done with more stable means.  I apologize for the warts it's left on the codebase.

**Comment by tbpassin** [comment]
That's a very generous thing to say!  And it's true that for testing short bits of code or getting some Python-related constant, say, you can do it with CTRL-B or with VR3.  But I still use the Console pane from time to time.  It's especially helpful if I want to be able to still see output after I switch to another node.

**Comment by edreamleo** [comment]
@gatesphere Hi Jacob. There is no need to apologize for this plugin! Instead, I should apologize for my ill humor yesterday.

No, the plugin isn't perfect. But where would Leo be if we demanded perfection? :-)

The plugin uses code.InteractiveInterpreter. Embedding this class is a stretch. But it's an interesting stretch. 

Not supporting "flush=True" might be a bug in python's code module.  Or not. But the plugin has done nothing grossly wrong in defining PyInterp.write.

Lastly, my irritation with the problem in the leo_excepthook function was unwarranted.  Adding "flush=True" to that method was just a mistake. The last thing I want to do is spray "flush=True" around Leo's print statements! Indeed, I am going to remove "flush" kwargs outside of leoserver.py.

In short, I apologize to you personally for my irritation yesterday. I have no plans to remove the python_terminal plugin.

**Comment by gatesphere** [comment]
Hi Edward,

No offense taken by the irritation -- it's not the first time this silly plugin has caused headaches, and I sincerely doubt it will be the last, if it is allowed to stay in the repo.  I absolutely agree with the content of your comments and frustration yesterday, and the 'presentation layer', so to speak, did not offend or harm my feelings.  Hell, I barely wrote any code for this -- it's mostly a hacked version of StackOverflow thread :)

I haven't used this plugin personally since 2016 or so.  I'm surprised that there are users out there, quite honestly.

There is certainly a better, less bug-prone way of doing this kind of thing.  I don't have the time to dive into that, specifically, at the moment, though.  I remain sorry for the pain this silly bit of code has caused.

Thanks.

**Comment by edreamleo** [comment]
@gatesphere I think this plugin is worth keeping on a "use at your own risk" basis :-)
